indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -419,6 +419,9 @@ def get_track_reviewing_state(self, track):
         else:
             return AbstractReviewingState.mixed
 
+    def get_track_reviews(self, track):
```

Comment:
We have the same list comprehension in `get_track_reviewing_state` though. So when deciding whether I want to have 2 lines of code repeated in 2 different places, I preferred to introduce the new method. If the majority believes it's better with 2 list comprehensions, fine by me!